webapi/tct-svg-html5-tests/html5-svg-app/testscripts/foreign_content_003-manual.feature

```diff
@@ -0,0 +1,6 @@
+Feature: html5-svg
+ Scenario: foreign content three rect svg in button
+   When launch "html5-svg-app"
+     And I go to "w3c/foreign_content_003-manual.html"
+     And I save the page to "foreign_content_003-manual"
```

Comment:
No need "manual" tag in feature file here, please follow the rule we talked before